17 error Error: Command failed with exit code 1: git remote update
17 error fatal: unable to access 'https://github.com/Hao-yiwen/pnpm-yiwen.git/': Failure when receiving data from the peer
17 error     at makeError (/Users/haoyiwen/Documents/web/pnpm-yiwen/node_modules/.pnpm/execa@5.0.0/node_modules/execa/lib/error.js:59:11)
17 error     at module.exports.sync (/Users/haoyiwen/Documents/web/pnpm-yiwen/node_modules/.pnpm/execa@5.0.0/node_modules/execa/index.js:186:17)
17 error     at Object.execSync (/Users/haoyiwen/Documents/web/pnpm-yiwen/node_modules/.pnpm/lerna@8.1.2/node_modules/lerna/dist/index.js:142:31)
17 error     at updateRemote (/Users/haoyiwen/Documents/web/pnpm-yiwen/node_modules/.pnpm/lerna@8.1.2/node_modules/lerna/dist/index.js:7780:18)
17 error     at isBehindUpstream (/Users/haoyiwen/Documents/web/pnpm-yiwen/node_modules/.pnpm/lerna@8.1.2/node_modules/lerna/dist/index.js:7770:3)
17 error     at VersionCommand.initialize (/Users/haoyiwen/Documents/web/pnpm-yiwen/node_modules/.pnpm/lerna@8.1.2/node_modules/lerna/dist/index.js:8061:57)
17 error     at VersionCommand.runCommand (/Users/haoyiwen/Documents/web/pnpm-yiwen/node_modules/.pnpm/lerna@8.1.2/node_modules/lerna/dist/index.js:1861:36)
